### PR TITLE
Added config schema for aerospike server 7.1

### PR DESCRIPTION
**Pull request by sud82**

### CODE REVIEW

File: json/aerospike/7.1.0.json [hunk 1]
```diff
@@ -0,0 +1,3557 @@
...
+                  "type": "string",
+                  "default": " ",
+                  "description": "",
+                  "dynamic": false,
+                  "enterpriseOnly": false
+                },
+                "default-read-touch-ttl-pct": {
+                  "type": "integer",
+                  "default": 0,
+                  "minimum": 0,
```
Comment (dwelch-spike) [inline json/aerospike/7.1.0.json]:
```suggestion
                  "minimum": -1,
```
The minimum value at the set level is -1. See cfg.c line 3824 https://github.com/citrusleaf/aerospike-server/compare/7.0.0.7...7.1.0.0-rc2#diff-0b637b2046b4f0541a331f7703c763756b27b54f4cff3c8a53f77ecdda57c124R3823

File: json/aerospike/7.1.0.json [hunk 2]
```diff
@@ -0,0 +1,3557 @@
...
+    "security": {
+      "type": "object",
+      "additionalProperties": false,
+      "enterpriseOnly": true,
+      "properties": {
+        "default-password-file": {
+          "type": "string",
+          "default": "",
+          "description": "",
+          "dynamic": true,
```
Comment (dwelch-spike) [inline json/aerospike/7.1.0.json]: Are you sure this is dynamic? I don't see any code to set it from info commands in RC2.

**Review by dwelch-spike — CHANGES_REQUESTED** [review]
I see a few differences between this and the server RC2 code. See comments. Otherwise this LGTM.

File: json/aerospike/7.1.0.json (hunk 1 above)
Comment (jdogmcsteezy) [reply json/aerospike/7.1.0.json:2157]: The schemas represent a valid aerospike.conf file. Internally, the server stores `-1` to indicate that `default-read-touch-ttl-pct` was not set by the user. `-1` is not a valid value in the aerospike.conf file OR an info call

Comment (jdogmcsteezy) [reply json/aerospike/7.1.0.json]: nvm, you are right

File: json/aerospike/7.1.0.json (hunk 2 above)
Comment (jdogmcsteezy) [reply json/aerospike/7.1.0.json]: ref: https://github.com/citrusleaf/aerospike-server-enterprise/blob/dfa163816c0da78556e5b7c167ff79cfa5987da7/as/src/base/security_info.c#L129

Comment (sud82) [reply json/aerospike/7.1.0.json]: Its static.

**Review by jdogmcsteezy — APPROVED** [review]
As long as the related changes are made in the generator, LGTM

**Review by dwelch-spike — APPROVED** [review]
LGTM

**Pull request merged**